flutter/ios/ci_scripts/ci_post_clone.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# abort if an error occurred
+set -e
+
+export LOG_PREFIX="[mobile_app_open]"
+cd ../../../
+echo "$LOG_PREFIX Current working directory is $PWD"
+# check if we are in the root directory of the repo, where the WORKSPACE file exists
+test -f WORKSPACE
+export ROOT_DIR=$PWD
+
+brew --version
+python3 --version
+
+echo "$LOG_PREFIX Install dependencies"
+export HOMEBREW_NO_AUTO_UPDATE=1
+export HOMEBREW_NO_INSTALL_CLEANUP=1
+brew install bazelisk
+bazel --version
+brew install protobuf
+protoc --version
+brew install cocoapods
+pod --version
+
+pip3 install --upgrade pip
+pip3 install \
+  numpy==1.21.5 \
+  absl-py==1.0.0
+
+echo "$LOG_PREFIX Install Flutter"
+export BUILD_HOME="$HOME"/build
+export FLUTTER_HOME="$BUILD_HOME"/flutter
+mkdir -p "$BUILD_HOME"
+test ! -d "$FLUTTER_HOME" && git clone --branch 2.10.5 --depth 1 https://github.com/flutter/flutter.git "$FLUTTER_HOME"
+export PATH="$PATH:$FLUTTER_HOME/bin:$HOME/.pub-cache/bin"
+dart pub global activate protoc_plugin
+
+echo "$LOG_PREFIX Build app"
+cd "$ROOT_DIR" && make
+cd "$ROOT_DIR"/flutter && flutter precache --ios
+cd "$ROOT_DIR"/flutter/ios && pod install
+cd "$ROOT_DIR"/flutter && flutter build ios --config-only integration_test/first_test.dart
```

Comment:
The build uploaded to testflight also launches the test.
We can probably differentiate release and test builds if we create 2 separate xcode cloud workflows and set different environment variables.

As an alternative we can just disable uploading to testflight and do it manually.
But I would prefer to have automatic uploads, they are nice to have.

Reply:
`Archive` action uploads resulting binary to testflight.
Since both `Test` and `Archive` actions use the same `ci_post_clone.sh` script, `Archive` uploads the binary that was built for testing.
If you install teslflight on your iphone and install `2.0.0 (17)` build from there, it will run benchmarks automatically, and will not allow you to interact with UI.

Reply:
Ok, I will fix that, thanks for the catch.